Library/Homebrew/formula.rb

```diff
@@ -1364,6 +1364,18 @@ def self.clear_installed_formulae_cache
     @installed = nil
   end
 
+  def self.comparison
```

Comment:
Since this is used for both Formulae and Casks, this should live somewhere else, and have a self-explanatory name.

Also, maybe add a test for this. This could even be simplified to `->(a) { [a.include?("/"), a] }` I think, but I'm not sure without a test.